editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -3480,7 +3468,7 @@ export const ComponentDescriptorKeepDeepEquality: KeepDeepEqualityCall<Component
     (descriptor) => descriptor.focus,
     createCallWithTripleEquals<Focus>(),
     (descriptor) => descriptor.inspector,
-    InspectorSpecKeepDeepEquality,
+    createCallFromIntrospectiveKeepDeep(),
```

Comment:
IIRC this is quite expensive, so we try to avoid it unless it's absolutely necessary